legacy/application/configs/conf.php

```diff
@@ -60,11 +60,11 @@ public static function loadConfig()
         $CC_CONFIG['cache_ahead_hours'] = $values['general']['cache_ahead_hours'];
 
         // Database config
-        $CC_CONFIG['dsn']['username'] = $values['database']['dbuser'];
-        $CC_CONFIG['dsn']['password'] = $values['database']['dbpass'];
-        $CC_CONFIG['dsn']['hostspec'] = $values['database']['host'];
         $CC_CONFIG['dsn']['phptype'] = 'pgsql';
-        $CC_CONFIG['dsn']['database'] = $values['database']['dbname'];
+        $CC_CONFIG['dsn']['hostspec'] = $values['database']['host'];
```

Comment:
Will need to add the port option here too

Reply:
I am unsure if this is easily doable, the legacy codebase totally ignored this field, and it would mean tracking doing every part of the code to fix this. I say that because there is a lot of code duplication and we need to tackle each one of them.

Reply:
Ok, this wasn't that difficult, I haven't tested this, but let's hope we won't have trouble with this. We will only notice bug if people change there database port anyway.